osu.Game/Graphics/Containers/TapToConfirmContainer.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Graphics.Containers
+{
+    public class TapToConfirmContainer : Container
```

Comment:
I wasn't so sure about adding a container like this, as I could only find a `HoldToConfirmContainer` which was used for hotkeys like this but this is not an instant one. If there is a better way to do this then please tell me.

Reply:
Even if this was required (which it isn't), it should be `abstract`.